extensions/wikia/AdEngine/js/DOMElementTweaker.js

```diff
@@ -0,0 +1,64 @@
+/*global define*/
+define('ext.wikia.adEngine.domElementTweaker', [
+	'wikia.log',
+	'wikia.document',
+	'wikia.window'
+], function (log, doc, win) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.slotTweaker',
+		rclass = /[\t\r\n]/g;
+
+	function removeClass(element, cls) {
+		var oldClasses,
+			newClasses = ' ' + element.className.replace(rclass, ' ') + ' ';
+
+		// Remove all instances of class in the className string
+		while (oldClasses !== newClasses) {
+			oldClasses = newClasses;
+			newClasses = oldClasses.replace(' ' + cls + ' ', ' ');
+		}
+
+		log(['removeClass ' + cls, element], 8, logGroup);
+		element.className = newClasses;
+	}
+
+	function isElement(obj) {
+		return obj instanceof HTMLElement;
+	}
+
+	function moveStylesToInline(element) {
+		var computedStyles,
+			styleName = "";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "styleName" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1854)